docker/alba_centos7_jenkins/Dockerfile

```diff
@@ -126,6 +126,14 @@ RUN git clone https://github.com/domsj/orocksdb.git \
 # be uniform on shared lib locations
 RUN cp /usr/local/lib/librocksdb.so* /lib64/
 
+#for now, instll ordma manually
+RUN yum -y install librdmacm-devel
+RUN git clone https://github.com/toolslive/ordma.git \
+    && cd ordma \
+    && git checkout tags/0.0.2 \
```

Comment:
That change is not really needed (the argument was wrongly ignored, and the field is set to `Closed` but it's only called with `Closed` as argument). So updating doesn't bring anything. meanwhile,
ordma is installable via opam. That would be an idea too.